src/EnergyPlus/BaseboardElectric.hh

```diff
@@ -57,6 +57,8 @@
 
 namespace EnergyPlus {
 
+    struct EnergyPlusData;
```

Comment:
So the main struct will live inside the EnergyPlus namespace.  Anywhere that we are using a forward declaration of this struct we need to put it inside the EnergyPlus namespace, not before it, and not inside another nested namespace.  This goes with any other lower level structs you may use as well.